### DISCUSSION TITLE
Documentation to SAML Azure

### DISCUSSION THREAD

**Discussion by sylbn** [test-results-reporter/reporter#182] [Q&A]
Hello,

Since the connection to the portal will become mandatory.
My company will need to have the SAML Azure portal connection documentation to be able to authorize using the tool.
Can you provide this to me ?

Regards,
Sylvain

**Comment by ASaiAnudeep** [comment]
Hello @sylbn,  Thank you for reaching out regarding the Azure portal connection.

We don't currently support Azure SAML. In the meantime I recommend creating a Microsoft account with your work email. This will allow you and your team to log in using the existing Microsoft authentication provider we have enabled. 

This will help all users under the same domain will be able to access the application uniformly, streamlining the login process and reduces the need for additional configuration and maintenance.

**Comment by sylbn** [comment]
Hello, we have already a Microsoft account and we use the SSO for several application.
I try to make testbeat approved by our security team so we can connect to the portal.
They ask about a documentation of how the MFA has been implemented, are you able to bring me this ?

**Reply by ASaiAnudeep** [reply]
As of now we don't have a documentation around it. I will share here once it is ready.
